docs/source/form/select/components/resource-select.md

```diff
@@ -123,6 +123,12 @@ When true, if the `resource` only returns one result the first time it is called
 
 When false or a function that returns false, the network request won't be made. Defaults to `true`.
 
+### `shouldFilterByAllOptions: boolean`
```

Comment:
i think the name of this prop is difficult to understand at a glance. I'm not entirely sure what to call it - maybe something like `shouldSearchClientSide`?

Reply:
+1 for `pageAll`